platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/PluginDependenciesSpecExtensions.kt

```diff
@@ -25,11 +25,27 @@ import org.gradle.plugin.use.PluginDependencySpec
  *
  * Visit the [Build Scan Plugin User Manual](https://docs.gradle.com/build-scan-plugin/) for additional information.
  *
- * By default, the applied plugin version will be the same as the one used by the `--scan` command line option.
+ * By default, the applied plugin version will be "3.16.2", which is the last version of the Gradle Enterprise plugin.
```

Comment:
❓ Why don't we use 3.17 for the Gradle Enterprise plugin as well and mark this as deprecated?